proc/proc.go

```diff
@@ -173,12 +173,12 @@ func parseKernelModuleLine(line string) (kernelModule, error) {
 
 	size, err := parseSize(parts[1])
 	if err != nil {
-		return kernelModule{}, fmt.Errorf("failed to parse size value: '%s'", parts[1])
+		return kernelModule{}, err
```

Comment:
Maybe we should keep the previous wording but also log the actual error, as this could help differentiate between all the different possible sources of these parsing errors.

Reply:
I'm not a fan of this message here, as it does not add value to the error.
The function `parseSize` returns `failed to parse size int value: %q", sizeStr` as an error.
https://github.com/open-telemetry/opentelemetry-ebpf-profiler/blob/7472b3a277f86c0d42683f2c0e23d6850dc71135/proc/proc.go#L203
And here the error is just repeated without adding additional value. Error messages should be generated as close as possible to the error, this makes it easier to manage and lookup if they happen.